lib/scrabble.rb

```diff
@@ -1,7 +1,14 @@
 class Scrabble
 
   def score(word)
-    1
+    letter_values = point_values
+    if word == "" || word == nil
+      0
+    else
+      word.split("").map do |letter|
```

Comment:
Like that you used map here! This method is a bit long though and could benefit from breaking this block out into its own method (or handle nil another way)